src/Numerical_aux/typesdef.f90

```diff
@@ -648,6 +648,9 @@ MODULE TypesDef
      REAL, allocatable                      :: forced_rupture_time(:,:)        !< forced rupture time at given fault node
      REAL, allocatable                      :: rupture_time(:,:)               !< rupture time at given fault node> used for VR ouput calculation
      REAL, allocatable                      :: dynStress_time(:,:)             !< time at which the shear stress is equal the dynamic stress
+     REAL, allocatable                      :: Yoffeonset(:,:)                  !< FL33 input parameter (rupture onset time of the Yoffe function, see Tinti et al., 2005)
+     REAL, allocatable                      :: YoffeTS(:,:)                     !< FL33 input parameter (ts, related with the acceleration time of the Yoffe function)
+     REAL, allocatable                      :: YoffeTR(:,:)                     !< FL33 input parameter (tr, related with the effective rise time of the Yoffe function)
```

Comment:
TS? TR? Could we have more expressive names?

Reply:
These are the naming used in Tinti 2005.
Note that these magic relations:

        ! Ts = time_acc*(1./1.27)
        DISC%DynRup%YoffeTS(:,:) =  DISC%DynRup%YoffeTS(:,:)/1.27
        ! Tr = rise_time_effective - 2.0*Ts;
        DISC%DynRup%YoffeTR(:,:) =  DISC%DynRup%YoffeTR(:,:) - 2.0 * DISC%DynRup%YoffeTS(:,:)
       
are not 100% precise. As far as I tested, the rise_time_effective formula seems ok, but the estimation of the acceleration time might diverge from 1.27*Ts.
I checked numerically, and found that the required ratio may vary between 1.2 and 1.4, as you can see in the following figure:
![Yoffe_127](https://user-images.githubusercontent.com/13536910/133979293-7a84b471-1cb0-434a-a6ad-6a507d8c813f.png)
But I guess the range 0.1 to 0.3 for t_acc/rise_time_effective is a typical range and in that range, 1.27 is a good approximation.

Reply:
wtf?

Reply:
Ok, that is hugly, but I m only following what was previously done. I bet the best would be to find analytically (e.g. with Maple) the maximum of this complicated function.

Reply:
Ok I understand that this problem is not easy to solve analytically. But before hard-coding a magic number like 1.27 (that is wrong) I'd rather let the user specify TS. In particular because the user draws physical conclusions from a wrong number!

Moreover, within easi, a user can easily divide 1.27 and get the same effect (but he's going to be aware of the caveat)

Reply:
yes you are right. What name would you give to the ts and tr parameters in easi?
What about YoffeTS and YoffeTR?

Reply:
How about tau_S and tau_R? That's closest to the paper which one has to read anyway, I guess.